README.md

```diff
@@ -34,14 +34,27 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+every semantic html tag has it's meaning. it makes it easier for us to markup a page and for other to read our code
+
```

Comment:
Very true, great answer. Another reason is accessibility, in some tech fields lack of semantic code can lead to accessibility legal action here is a great resource for more info ->https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML